content/docs/command-reference/status.md

```diff
@@ -209,3 +214,27 @@ Preparing to collect status from s3://dvc-remote
 
 The output shows where the location of the remote storage is, as well as any
 differences between the <abbr>cache</abbr> and `storage` remote.
+
+## Example: Import Stage
+
+Let us import a file `data.csv` from a <abbr>DVC repository</abbr> stored
+locally to our current repository using `dvc import`.
+
+```dvc
+$ dvc import /home/user/path/ data.csv
+Importing 'data.csv (/home/user/path/)'
+-> 'data.csv'
+```
+
+This is called a <abbr>import stage</abbr>. If the orginal file or directory is
+changed then running `dvc status` will show `update available` as output.
+
+```dvc
+$ dvc status
+data.csv.dvc:
+	changed deps:
+		update available:   data.csv (/home/user/path/)
+```
+
+This shows that orginal file or directory is changed. The import stage can be
+brought to lates version by using `dvc update`.
```

Comment:
```suggestion
The imported data can be brought to its latest version by using `dvc update`.
```
that's good enough since the previous paragraph already explained about the source being changed.

But definitely spell checker please (typos: "orginal", "lates") 🙂

Reply:
Thanks for your inputs and plugin suggestion as I use VSCode too. 
I changed the location to `different/repo/location` in `dvc status` output also.